mongo/tests/conftest.py

```diff
@@ -0,0 +1,136 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import subprocess
+import os
+import logging
+import time
+
+import pymongo
+import pytest
+
+from datadog_checks.mongo import MongoDb
+
+from . import common
+
+log = logging.getLogger('conftest')
+
+
+@pytest.fixture
+def aggregator():
```

Comment:
With the new dev tooling (datadog-checks-dev) I think you can remove these lines and the aggregator is still an available fixture.